Sources/Orbit/Components/Alert.swift

```diff
@@ -191,6 +195,10 @@ public struct Alert<Content: View>: View {
             ? .secondary
             : .status(status, subtle: true)
     }
+
+    var isHeaderEmpty: Bool {
```

Comment:
I think I got rid of this on purpose - does it actually do anything if all the parts resolve to `EmptyView` anyway?

Reply:
It seems to be required, because nested stack views do not seem to have the recursive logic for evaluating whether the whole content tree consists purely of `EmptyView`s up to the root stack. This means, there would be empty "stacks" that take part in parent stack spacing.